pkg/internal/gomocks/client/introduce/mocks.go

```diff
@@ -1,63 +0,0 @@
-// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Why delete

Reply:
we do not use it anymore, I can't even generate it.